opentelemetry-sdk/CHANGELOG.md

```diff
@@ -370,6 +372,29 @@ let processor = BatchSpanProcessor::builder(exporter)
     .build();
 ```
 
+- **Breaking**: The `PeriodicReader` no supports configuration of export timeout using
+  `with_timeout` API method. 
+  Timeout handling is now the responsibility of the exporter.
+
+  For example, in the OTLP Metrics exporter, the export timeout can be configured using:
+  - The environment variables `OTEL_EXPORTER_OTLP_TIMEOUT` or `OTEL_EXPORTER_OTLP_METRICS_TIMEOUT`.
+  - The `opentelemetry_otlp` API, via `.with_tonic().with_timeout()` or `.with_http().with_timeout()`.
+ 
+  Before:
+```rust
+let reader = PeriodicReader::builder(exporter)
+    .with_interval(Duration::from_secs(30))
+    .with_timeout(Duration::from_secs(10)) // Previously configurable timeout
+    .build();
+```
+
+ After:
+```rust
+let reader = PeriodicReader::builder(exporter)
```

Comment:
this maybe confusing as we now don't require users to build reader. better to skip showing before/after completely here.